modules/core/src/main/java/org/apache/ignite/internal/processors/platform/compute/PlatformBalancingSingleClosureAffinityTask.java

```diff
@@ -31,6 +31,8 @@
 
 /**
  * Interop single-closure task with node balancing.
+ *
+ * NOTE: This class is not used, but preserved for compat during rolling upgrades.
```

Comment:
This class doesn't transfer through the wire, why should we take care of it during RU?

Reply:
Good catch, fixed